### PR TITLE
may-change-logs

### PR DESCRIPTION
**Pull request by kghamilton89**
Omitted issues: 5283, 5312, 5273, 2346 (charts)

### CODE REVIEW

**Comment by rodionos** [comment]
List of issues for May changelog is specified in 5344 in redmine. Please use it as source.

**Comment by kghamilton89** [comment]
I guess I must have reordered the chart in RedMine before I exported. All issues were there for ATSD but incorrectly ordered.

File: changelogs/2018_05/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+5324 | rule engine    | Bug     | Change [`randomItem`](../../rule-engine/functions-random.md#randomitem) function to return the same object type as the argument.
+5321 | core           | Feature | Implement **Settings** > **Schema** > **Series Consistency Check** page.
+5319 | rule engine    | Bug     | Fix an infinite loop when monitoring rule errors in rule engine itself.
+5318 | nmon           | Bug     | [`nmon`](../../integration/nmon/README.md#nmon) property record is stale and is not updated.
+5315 | message        | Bug     | Fix incorrect [wildcard](../../search/entity-search.md#wildcards) match on the message search page.
+5313 | client         | Bug     | [Python Client](https://github.com/axibase/atsd-api-python): [Message model](https://github.com/axibase/atsd-api-python#inserting-data) standardized.
+5310 | message        | Feature | Modify filter processing for tag wildcards and empty tag values on the message search page.
+5309 | api-rest       | Bug     | `NullPointerException` raised on [message insert](../../api/network/message.md#message-command) when one of the tags is set to `null`.
```
Comment (rodionos) [inline changelogs/2018_05/README.md]: Link should be set to rest api, not network api

File: changelogs/2018_05/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## ATSD
+
+**Issue**| **Category**    | **Type**    | **Subject**
+-----|-------------|---------|----------------------
+5340 | statistics     | Bug     | Server error raised when accessing statistics page for series with [NaN](../../sql/README.md#not-a-number-nan) values.
+5338 | export             | Bug     | Series tags validation error on the export page.
+5337 | client | Feature | Value filter added for [Python Client](../../README.md#axibase-time-series-database-client-for-python).
+5334 | api-rest       | Bug     | Properties [delete query](../../administration/data_retention.md#deleting-properties) returns zero counter even after deleting some records.
+5333 | client         | Bug     | [Python Client](../../README.md#axibase-time-series-database-client-for-python) tests are failing due to time precision.
```
Comment (rodionos) [inline changelogs/2018_05/README.md]: https://github.com/axibase/atsd-api-python - correct link to python client

File: changelogs/2018_05/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Monthly Change Log: May 2018
+
+## ATSD
+
+**Issue**| **Category**    | **Type**    | **Subject**
+-----|-------------|---------|----------------------
+5340 | statistics     | Bug     | Server error raised when accessing statistics page for series with [NaN](../../sql/README.md#not-a-number-nan) values.
+5338 | export             | Bug     | Series tags validation error on the export page.
+5337 | client | Feature | Value filter added for [Python Client](../../README.md#axibase-time-series-database-client-for-python).
```
Comment (rodionos) [inline changelogs/2018_05/README.md]: Link to python client: https://github.com/axibase/atsd-api-python

File: changelogs/2018_05/README.md (hunk 2 above)
Comment (kghamilton89) [reply changelogs/2018_05/README.md]: Sorry this was a search and replace error

File: changelogs/2018_05/README.md (hunk 3 above)
Comment (kghamilton89) [reply changelogs/2018_05/README.md]: likewise

File: changelogs/2018_05/README.md (hunk 1 above)
Comment (kghamilton89) [reply changelogs/2018_05/README.md]: REST API linked.

**Comment by kghamilton89** [comment]
Grammar was fine, articles were correct. Generally in something like a Change Log, it's perfectly acceptable to omit most articles, it's a type of shorthand. I typically do it, so I went through and removed articles from where maybe you or I had added them for standardization purposes. 

For reference the [Chrome Release Notes](https://chromereleases.googleblog.com/) almost never contain articles, even where they might be required should the context be more standard. Same is true in [Linux Change Logs](https://kernelnewbies.org/LinuxChanges#Linux_4.15.File_systems).

Articles are just a nightmare (even the sentence I used above could easily be written, "the grammar was fine, the articles were correct."

**Pull request merged**